### PR TITLE
Fixes #9434: Create a script to generate reports

### PR DESCRIPTION
**Pull request by ncharles**

### CODE REVIEW

File: contrib/load-database/load-database.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python
+
+hostname = 'localhost'
+username = 'rudder'
+password = 'Normation'
+database = 'rudder'
+
```
Comment (peckpeck) [inline contrib/load-database/load-database.py]: To make it compatible with python 3, add here: 

from __future__ import print_function

File: contrib/load-database/load-database.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+for nodeid, nodeconfigid, begindate, configuration in cur.fetchall():
+    skew = random.random()*60
+    reportDate = (datetime.datetime.now()+datetime.timedelta(0, skew)).replace(microsecond=0)
+    d = json.loads(configuration)
+    for rules in d['rules']:
+        for directives in rules['directives']:
+            for components in directives['components']:
+                for value in components['values']:
+                    write.execute('insert into ruddersysevents(executiondate, nodeid, directiveid, ruleid, serial, component, keyvalue, executiontimestamp, eventtype, policy, msg) values (%s, %s, %s, %s, %s, %s, %s , %s , %s , %s, %s)', (reportDate, nodeid, directives['directiveId'], rules['ruleId'],  rules['serial'], components['componentName'], value, reportDate, 'result_success', '', 'Dummy reports for load test'))
+                    print reportDate, nodeid, directives['directiveId'], rules['ruleId'],  rules['serial'], components['componentName'], value, reportDate, 'result_success', '', 'Dummy reports for load test'
```
Comment (peckpeck) [inline contrib/load-database/load-database.py]: Then print is a function and needs parens: print("xxx")

File: contrib/load-database/load-database.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        if directives['directiveId'].startswith('common-'):
+            write.execute('insert into ruddersysevents(executiondate, nodeid, directiveid, ruleid, serial, component, keyvalue, executiontimestamp, eventtype, policy, msg) values (%s, %s, %s, %s, %s, %s, %s , %s , %s , %s, %s)', (reportDate, nodeid, directives['directiveId'], rules['ruleId'],  rules['serial'], 'common', 'EndRun', reportDate, 'log_info', '', 'End execution with config [' + nodeconfigid + ']'))
+            print reportDate, nodeid, directives['directiveId'], rules['ruleId'],  rules['serial'], 'common', 'EndRun', reportDate, 'log_info', 'common', 'End execution with config [' + nodeconfigid + ']'
```
Comment (peckpeck) [inline contrib/load-database/load-database.py]: Here too

File: contrib/load-database/load-database.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+
+print 'Commiting reports'
```
Comment (peckpeck) [inline contrib/load-database/load-database.py]: idem

File: contrib/load-database/load-database.py (hunk 1 above)
Comment (peckpeck) [inline contrib/load-database/load-database.py]: future with underscores

**Comment by peckpeck** [comment]
Do you want to make some changes or should i merge it as is ?

**Comment by ncharles** [comment]
PR updated

**Comment by ncharles** [comment]
PR updated

**Pull request merged**